hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -609,11 +603,8 @@ private void writeChunkToContainer(ChunkBuffer chunk) throws IOException {
         }
         return e;
       }, responseExecutor).exceptionally(e -> {
-        if (LOG.isDebugEnabled()) {
-          LOG.debug(
-              "writing chunk failed " + chunkInfo.getChunkName() + " blockID "
-                  + blockID + " with exception " + e.getLocalizedMessage());
-        }
+        LOG.debug("writing chunk failed {} blockID {} with exception{}",
```

Comment:
Nit: lost space in message.

```suggestion
        LOG.debug("writing chunk failed {} blockID {} with exception {}",
```